setup.py

```diff
@@ -1,46 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
My python/setuptools knowledge isn't that deep, but my only real project I produce was done the same way without any of the stuff in `setup.py`. In the end I decided to keep a minimal [setup.py](https://github.com/dbnicholson/ostree-push/blob/master/setup.py) for familiarity and because Debian wanted it when I was trying to get it packaged. I don't think either of those concerns make sense here, so I'm totally fine if it goes.